dipy/tracking/streamline.py

```diff
@@ -89,6 +90,40 @@ def center_streamlines(streamlines):
     return [s - center for s in streamlines], center
 
 
+def deform_streamlines(streamlines, deformField, streams2grid, grid2world):
+    """ Apply deformation field to streamlines
+
+    Parameters
+    ----------
+    streamlines : list
+        List of 2D ndarrays of shape[-1]==3
+    deformField : 4D numpy array
+        x,y,z displacements stored in volume, shape[-1]==3
+    streams2grid : array, (4,4)
```

Comment:
Is this the affine from the streamlines to the acquisition grid?

Reply:
this takes streamlines from voxmm space to image grid space.  So for instance if both streamlines and the image were LAS and vox dimensions were 2,2,2 it would effectively be the transformation that divides all streamline points by 2.